doc/source/whatsnew/v0.21.0.txt

```diff
@@ -325,7 +325,7 @@ Performance Improvements
 
 - Improved performance of instantiating :class:`SparseDataFrame` (:issue:`16773`)
 - :attr:`Series.dt` no longer performs frequency inference, yielding a large speedup when accessing the attribute (:issue:`17210`)
-
+- `Timestamp.microsecond` no longer re-computes on attribute access.
```

Comment:
add the PR number as the issue. we *always* want a reference to an entry in whatsnew (issue preferd, or PR if not avail).

Reply:
either use double-backticks, or `` :attr:`Timestamp.microsecond` `` (in general prefer the api references). I don't think this API reference will actually work ATM as ``Timestamp`` is not defined in api.rst (so separate issue to rectify that).